### PR TITLE
Prd add issuer componenet

### PR DESCRIPTION
**Pull request by rubenmodamioin2**
Prd add issuer componenet

### CODE REVIEW

**Review by bialask-daiteap — CHANGES_REQUESTED** [review]

File: ionos_prd/issuer/templates/dome-issuer-keycloak-admin-sealed-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: dome-issuer-keycloak-admin-secrets
+  namespace: in2
+spec:
+  encryptedData:
+    password: AgDDZqo/Uh1hkpYTggeGXeEWbrDu85ld1OrBBF97HkeFB0rDplNMvSLJGVinS6aMTjq9EFcKLxPIvrhfrc88rRfo9KxVSa7A3q6mrskHBCbuPM0kYjnuNA4bT/907bIuPHgY4OdbmKsdkzYi/JOf1uQHs7/A9cOGYnkGW/t+tcfkCSoeCXTl7ISnwBlTKGh2m7kWDTgnCm4/wFtyZplf2Sq/khLMl5gHAEqU66lBrech8nmPCgZ6pWtj38CkP7rjSB9RfskXV07jCWzRfRaxfvRPJ7qVanU9WRkScXtoC60qX4uyVeIAh5s2R0ngjFug0GN6PtLfgOgSQnvPlDIArduAYuDmWPhBKvkD9GD0B/GwG0UCdPmIK0OCylPCnM46ffnd/FmWbQ6D/tbvF2pqbK11aqjnbe1roCLwmvoOeYcMYF5JknGOgsus6f5PRZJA2At+6NTXcJZa4oMw68spdmg7O55t64M+s7gqU999q9ZkZqqFWBf2WL27wDdrUozhvnqfYEJ3ClPV48/EStvhNYvcC4q1KDikW4XbBt/ikdebMBMNam3LN2lSiXV4RYVnYPlM6m4+7Ez2sJQI6jbhBaUhZcAJWoggPSPVxuIsPzU6d7ca5dXQ/eT/naMSE+ttmP4xCyVgimCjyE/IwkwsslHSLXfBY4gOpGdNoeSxpeqojgKgdywARzH2HWYUKvEQC0/2PEdtHqx1E8gmtQU=
```
[the remark this answers is not in the document]
Comment (rubenmodamioin2) [reply ionos_prd/issuer/templates/dome-issuer-keycloak-admin-sealed-secret.yaml:10]: It's fine to put all the secrets in one file, but there's one thing I'd like to discuss regarding this: in the dev PR, which is linked here https://github.com/DOME-Marketplace/dome-gitops/pull/314 , as you can see, both in dev and SBX they are in several files. If it seems good to you, I can create a PR to group them into a single file starting with SBX and then proceeding to dev2.

[the remark this answers is not in the document]
Comment (bialask-daiteap) [reply ionos_prd/issuer/templates/dome-issuer-keycloak-admin-sealed-secret.yaml:10]: ok let's keep consistency. But for the future please keep them in one file / resource if any changes are planned

[the remark this answers is not in the document]
Comment (rubenmodamioin2) [reply ionos_prd/issuer/templates/dome-issuer-keycloak-admin-sealed-secret.yaml:10]: Okay ill start with SBX

[the remark this answers is not in the document]
Comment (rubenmodamioin2) [reply ionos_prd/issuer/templates/dome-issuer-keycloak-admin-sealed-secret.yaml:10]: I've created the PR on SBX: https://github.com/DOME-Marketplace/dome-gitops/pull/405

[the remark this answers is not in the document]
Comment (rubenmodamioin2) [reply ionos_prd/issuer/templates/dome-issuer-keycloak-admin-sealed-secret.yaml:10]: I've created the [PR](I've created the PR on SBX: https://github.com/DOME-Marketplace/dome-gitops/pull/405l) on SBX

File: applications_prd/release1/in2/issuer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
```
[the remark this answers is not in the document]
Comment (rubenmodamioin2) [reply applications_prd/release1/in2/issuer.yaml:3]: https://github.com/DOME-Marketplace/dome-gitops/pull/314

[the remark this answers is not in the document]
Comment (bialask-daiteap) [reply applications_prd/release1/in2/issuer.yaml:3]: Ok thank you

**Comment by bialask-daiteap** [comment]
The MR is fine. Please move the application from /release1 to /release2 folder

**Comment by rubenmodamioin2** [comment]
I created the release2 folder and now the issuer is inside the release 2 folder

**Pull request merged**